### DISCUSSION TITLE
Can't interact to anything in a repository as a user

### DISCUSSION THREAD

**Discussion by ghost** [community/community#21998] [Other Feature Feedback, Questions, & Ideas]
<p>“An owner of this repository has limited the ability to comment to users that have contributed to this repository in the past.”</p>
<p>Why is this a thing? I can’t comment, nor able to open an issue in a repository. In other repos it’s perfectly fine.</p>
<p>This is what it says when I try to open an issue:</p>
<p>"Documentation has changed since you last contributed 14 days ago. Take a look before submitting an issue:</p>
<p>Contributing guidelines<br>
Last updated yesterday"</p>

**Comment by jfullstackdev** [comment]
<p>you mean you contributed but can’t comment because of the limitation ? can you send the exact link of the repo so we can take a look</p>

**Comment by airtower-luna** [comment, marked as answer]
<aside class="quote no-group" data-username="pc00per" data-post="1" data-topic="244572">
<div class="title">
<div class="quote-controls"></div>
 pc00per:</div>
<blockquote>
<p>Why is this a thing? I can’t comment, nor able to open an issue in a repository. In other repos it’s perfectly fine.</p>
</blockquote>
</aside>
<p>It’s a moderation feature: <a href="https://docs.github.com/en/communities/moderating-comments-and-conversations/limiting-interactions-in-your-repository" class="inline-onebox">Limiting interactions in your repository - GitHub Docs</a></p>
<p>Obviously I can’t know why the owner of some repository chose to enable those limits, but usually it is to get some quiet if someone is target of things like spam, a flame war, or harassment.</p>

**Reply by KoghaandSooga** [reply]
It's pure censorship.  I wanted to explain that Edge no longer supports the extension and I got that message. I wonder if perhaps the owner got mad at a bunch of complaints instead of addressing it like an adult he threw a tantrum like a snowflake having a digital meltdown on Twitter or Reddit over something stupid.

**Reply by fillwithjoy1** [reply]
@KoghaandSooga It's not easy being the repo maintainer when 80% of issues are from people who somehow make the most rudimentary mistakes and blames that your repo is the issue. I can see why he would do that, and saying that he's "having a digital meltdown" is just not nice. Just remember, **he made the source code for free, for the community**

**Comment by ghost** [comment]
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/TeamNewPipe/NewPipe">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/TeamNewPipe/NewPipe" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/83843264/181068143-a221d544-b0a2-498a-b1fb-85dbf0f1f045.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/a/3a4b7f0b530fdd8b8bc1b3d32b1cc5bd87363a5f_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/a/3a4b7f0b530fdd8b8bc1b3d32b1cc5bd87363a5f_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/a/3a4b7f0b530fdd8b8bc1b3d32b1cc5bd87363a5f.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/a/3a4b7f0b530fdd8b8bc1b3d32b1cc5bd87363a5f_2_10x10.png"></div>

<h3><a href="https://github.com/TeamNewPipe/NewPipe" target="_blank" rel="noopener nofollow ugc">GitHub - TeamNewPipe/NewPipe: A libre lightweight streaming front-end for...</a></h3>

  <p>A libre lightweight streaming front-end for Android. - GitHub - TeamNewPipe/NewPipe: A libre lightweight streaming front-end for Android.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
<p>
I mean, I did opened one bad issue, which I regret. But I don’t think that’s the reason I got locked on this repo.</p>

**Reply by fillwithjoy1** [reply]
literally 1984

**Reply by fillwithjoy1** [reply]
The bad issue you opened is the reason why you got locked out. I suppose the team has no tolerance, you know they just want someone to provide all the **helpful** info so that they can burrow down to the issue. A bad issue just wastes time, since the team cannot pinpoint the cause. I'm sure you are a good person that tries to create helpful issues, but you made an oopsie one time which happens to all of us. Just remember why good issues help the team. You could go to the discussions page and comment to be unlocked, hopefully someone unlocks you after they rant about the bad stuff you did.

**Comment by jfullstackdev** [comment]
<p>unfortunately, too many issues, I cannot see the exact issue you opened,</p>

**Comment by DO2SN** [comment]
I have the same issue... Tried to start a discussion but "An owner of this repository has limited the ability to comment from new users. ". Can not interact with anything. Maybe after this first comment here, it will work. 

so hello world

**Reply by fillwithjoy1** [reply]
I can possibly help you, what you needing to post?

**Reply by fillwithjoy1** [reply]
I got the issue you were trying to post, but it disappeared here?

**Reply by DO2SN** [reply]
Yes. Thanks a lot for your help. Reason was that there is a kind of "time block" for new users. Now everything works fine. So I deleted my post.

**Reply by KevBer1** [reply]
How long does it take until new users are allowed to write comments?

**Reply by fillwithjoy1** [reply]
> How long does it take until new users are allowed to write comments?

Being 5 days old doesn't help

**Comment by NIKITADAVID0V** [comment]
Yes, this is a problem. The owner of this repository has limited the ability to open problems from new users.

**Comment by KoghaandSooga** [comment]
That's essentially the definition of a 'digital meltdown' where someone doesn't want the slightest big of (valid) criticism. Sure they released the product for free but that is still no excuse to be a jerk and in fact PAID products most support acts like jerks where you have to go thru hell and back to get any help...............and that's if you actually get any help.
________________________________

**Comment by waifucon** [comment]
Two possibilities

1. Blocked by the owner of the repo
2. The owner has set some limits due to trolling & harassmrnts.

**Comment by jinz24** [comment]
How can we comment if we have been limited?

**Reply by waifucon** [reply]
you couldn't.
